content/articles/faq-ssl-certificates.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: SSL Certificates Frequently Asked Questions
+excerpt: This is a collection of frequently asked questions about the SSL certificates offered by DNSimple.
+categories:
+- SSL Certificates
+---
+
+# SSL Certificate FAQ
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## Prices and Subscription
+
+<div class="section-faq" markdown="1">
+1.  #### What is the price of an SSL certificate?
+
+    The price depends on the certificate type. To learn more visit the [SSL Certificates](/articles/ssl-certificates/) article.
+
+1.  #### I want to purchase an SSL certificate. What is the DNSimple subscription?
+
+    DNSimple is a subscription-based service. In order to access our domain services, you need an active subscription to our platform.
```

Comment:
DNSimple is a subscription-based service. You need an active subscription in our to use our platform.